apps/web/modules/auth/lib/authOptions.ts

```diff
@@ -203,11 +203,17 @@ export const authOptions: NextAuthOptions = {
         if (!user.emailVerified && !EMAIL_VERIFICATION_DISABLED) {
           throw new Error("Email Verification is Pending");
         }
+        await updateUserByEmail(user.email, { lastLoginAt: new Date() });
         return true;
       }
       if (ENTERPRISE_LICENSE_KEY) {
-        return handleSSOCallback({ user, account });
+        const result = await handleSSOCallback({ user, account });
```

Comment:
correct camelcase would be `handleSsoCallback`
(it's not part of the PRs new code but let's fix it :-) )